.github/workflows/tests.yml

```diff
@@ -145,3 +145,6 @@ jobs:
 
       - name: Run ${{matrix.service}} Tests
         run: docker compose exec -T appwrite test /usr/src/code/tests/e2e/Services/${{matrix.service}} --debug
+
+      - name: Run ${{matrix.service}} Shared Tables Tests
+        run: _APP_DATABASE_SHARED_TABLES=database_db_main docker compose exec -T appwrite test /usr/src/code/tests/e2e/Services/${{matrix.service}} --debug
```

Comment:
Can we document this somewhere?

Reply:
As in, the syntax/ format of this variable

Reply:
Yep, will make more sense in cloud